src/pretix/plugins/banktransfer/payment.py

```diff
@@ -14,19 +15,22 @@ class BankTransfer(BasePaymentProvider):
 
     @property
     def settings_form_fields(self):
+        form_field = I18nFormField(
+            label=_('Bank account details'),
+            widget=I18nTextarea,
+        )
+        form_field.widget.enabled_langcodes = self.event.settings.get('locales')
         return OrderedDict(
-            list(super().settings_form_fields.items()) + [
-                ('bank_details',
-                 forms.CharField(
-                     widget=forms.Textarea,
-                     label=_('Bank account details'),
-                 ))
-            ]
+            list(super().settings_form_fields.items()) + [('bank_details', form_field)]
         )
 
     def payment_form_render(self, request) -> str:
         template = get_template('pretixplugins/banktransfer/checkout_payment_form.html')
-        ctx = {'request': request, 'event': self.event, 'settings': self.settings}
+        ctx = {
+            'request': request,
+            'event': self.event,
+            'details': LazyI18nString(self.settings.get('bank_details')),
```

Comment:
You could use ``self.settings.get('bank_details', as_type=LazyI18nString)`` here :)